<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.3.2 (Python 3.9.6 on darwin)" generated="2025-07-14T17:17:07.697549" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/Users/romeobajadajr/.cursor/extensions/robocorp.robotframework-lsp-1.13.0-universal/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-07-14T17:17:07.706419" elapsed="15.140156"/>
</kw>
<status status="PASS" start="2025-07-14T17:17:07.706177" elapsed="15.141470"/>
</test>
<status status="PASS" start="2025-07-14T17:17:07.698056" elapsed="15.150712"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-07-14T17:17:07.695782" level="WARN">Error in file '/Users/romeobajadajr/.cursor/extensions/robocorp.robotframework-lsp-1.13.0-universal/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
